exercises/pangram/instructions.md

```diff
@@ -0,0 +1,8 @@
+# Instructions
+
+Your task is to figure out if a sentence is a pangram.
+
+A pangram is a sentence using every letter of the alphabet at least once.
+It is case insensitive, so it doesn't matter if a letter is lower-case (e.g. `k`) or upper-case (e.g. `K`).
+
+For this exercise we only use the basic letters used in English: `a` to `z`.
```

Comment:
```suggestion
For this exercise we only use the basic letters used in the English alphabet: `a` to `z`.
```

Reply:
Thanks, yes. I think that's a good call.